www/_blog/2021-11-28-postgrest-9.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: 'PostgREST v0'
+description: Kicking off launch week by highlighting the communities around Supabase.
+author: steve_chavez
+author_url: https://github.com/steve-chavez
+author_image_url: https://github.com/steve-chavez.png
+authorURL: https://github.com/steve-chavez
+image: launch-week-three/community-day/supabase-oss.jpg
+thumb: launch-week-three/community-day/supabase-oss.jpg
```

Comment:
@MildTomato - could we please do a PostgREST 9 OG like you did for pg 14?

Reply:
adding now

Reply:
I saw this was resolved - but it looks like it's still pending?